components/eam/src/physics/cam/cloud_diagnostics.F90

```diff
@@ -155,9 +155,9 @@ subroutine cloud_diagnostics_init()
     call addfld ('TGCLDCWP',horiz_only,    'A',wpunits,'Total grid-box cloud water path (liquid and ice)', &
```

Comment:
This one can use the name 'atmosphere_mass_content_of_cloud_condensed_water', which includes liquid and ice.